common/src/main/java/com/wynntils/utils/JsonUtils.java

```diff
@@ -31,4 +31,12 @@ public static boolean getNullableJsonBoolean(JsonObject json, String key) {
 
         return jsonElement.getAsBoolean();
     }
+
+    public static JsonObject getNullableJsonObject(JsonObject json, String key) {
```

Comment:
I expected this method to return null from the name. I can't find a better name though.

Reply:
I thought all these JsonUtils method did the same (and returned a valid object) but I see now that the String version returns null instead of an empty string. I think I should fix that. And maybe then we can rename the methods, either just drop "Nullable" completely, or try weaving in a "safe" or something like that, e.g. `getJsonObjectSafe` or such....